README.md

```diff
@@ -1,2 +1,293 @@
 # Zone101
 My first Repository
+import java.util.Random;
+import java.text.Format;
+import java.text.NumberFormat;
+public class Ticket {
+
+	private String name, notice;
+	private int speed;
+	private int speedLimit;
+	private boolean workZone;
+	private boolean schoolZone;
+	private double fine = 0;
+
+	
+	public Ticket() {
+		this.name= name;
+		this.speed = speed;
+		this.speedLimit= speedLimit;
+	
+	}
+	/**
+	 * Constructor that takes data as input and sets them.
+	 * @param name the name of the violator.
+	 * @param speed the speed of the violator.
+	 * @param speedLimit the speedLimit of the region.
+	 * @param workZone the work zone region where the driver passed the speed limit.
+	 * @param schoolZone the school zone region where the driver passed the speed limit. 
+	 */
+// Correction switch the workZone and school zone order 
```

Comment:
The constructor was having issues when I compared it with a test drive. But it started working just fine when I switched the work and school zone.